README.md

```diff
@@ -6,13 +6,41 @@ Whenever you're starting a project, there are about a million questions that nee
 Usage
 -----
 
+There are three ways to generate a project using this template:
+
+- Interactively using the initialization guide
+- Automatically by initializing with default values
+- Downloading the latest release and editing files manually
+
+### Interactive project generation
+
+Run the following in a bash terminal, substituting `<directory>` with the directory you wish to create your project in:
+
+```bash
+git clone https://github.com/zambezi/project.git <directory> && <directory>/init
+```
+
+This will run the guide, which will take you through the process. Once it's done, cd in to `<directory>` and behold your glorious new project!
+
+### Automatic project generation
+
+This is similar to interactively generating a project, but you also add the `--default` flag with the script, like so:
+
+```bash
+git clone https://github.com/zambezi/project.git <directory> && <directory>/init --default
+```
+
+This will use default values when generating files, and is good for when you plan to add more information later.
+
+### Manually creating a project
+
 1. [Download the latest release](../../releases/latest)
 2. Extract the downloaded archive
 3. Follow the checklist below
```

Comment:
This should probably change to say:

3. Follow the checklist below – or – run `./init` in the directory where you extracted the archive